src/visual/VisualWorkflow.js

```diff
@@ -13,11 +13,15 @@ export default class VisualWorkflow extends VisualGroup {
     super(_.defaultsDeep(opts, {
       attrs: {
         '.label': {
-          text: opts.step.type,
+          text: `${opts.step.type} ${opts.step.name}`,
```

Comment:
Is it really necessary to see a step type on a diagram?

Reply:
There can be multiple subWorkflows in real WDL examples. So it's better to show name to be able to recognise one subWorkflow from other and show it's type to recognise subWorkflow visualisation from task visualisation